index.html

```diff
@@ -0,0 +1,123 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Memory Game</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./index.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-3" id="Moves">
+                <div class="alert alert-info" id="player-moves">
+                    Moves: 
+                </div>
+            </div>
+
+           <div class="col-sm-4" id="timer-panel">
+               <div class="alert alert-info" id= "timer">
+                Playtime:
+               </div>
+               
+           </div>
+
+           <div class="col-sm-5" id="status-panel">
+            <div class="alert alert-info" id= "status">
+             Welcome
+               </div>
+           </div>
+        </div>
+    </div>
+
+    <div class="container">
+        <button type="button"  class="btn btn-primary" id="start">Start</button>
+    </div> 
+
+
+    <div class="container" id="cards">
+        <div class="row">
+            <div class="col">
+                <div class="card color-hidden" onclick="onCardClicked(event)"></div>
+            </div>
+            <div class="col">
+                <div class="card color-hidden" onclick="onCardClicked(event)"></div>
+            </div>
+            <div class="col">
+                <div class="card color-hidden" onclick="onCardClicked(event)"></div>
+            </div>
+            <div class="col">
+                <div class="card color-hidden" onclick="onCardClicked(event)"></div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col">
+                <div class="card color-hidden" onclick="onCardClicked(event)"></div>
```

Comment:
el onclick="" sería mejor que lo hagas directamente en el archivo de javascript, la regla es que lo que tiene que ver con manipular elementos siempre esté explícito dentro de la lógica de javascript, y no disperso en varios archivos.

Reply:
hola, estuve añadiendo unas cositas al memotest, trate de sacar este onclick del html creando un id="card" en el div, luego guardandolo en una variable de el archivo js, dandole el onclick y luego que ejecute la funcion y no funciona :(

Reply:
Ahora si cree una const global para que me diera todo los elementos con la clase en css 'card', luego hice un for para recorrer todos esos elementos con un addEventListener que al hacer click ejecute la funcion onCardClicked